flask_iiif/api.py

```diff
@@ -152,9 +152,13 @@ def resize(self, dimensions, resample=None):
             width = max(1, int(real_width * percent))
             height = max(1, int(real_height * percent))
 
-        # Check if it is `,h`
-        elif dimensions.startswith(","):
-            height = int(dimensions[1:])
+        # Check if it is `,h` or `^,h`
```

Comment:
minor: maybe add a clarifying comment on top about our (wrong) handling of `^`, which is basically the same as if it wasn't there.